lib/aur-ninja

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+[[ -v AUR_DEBUG ]] && set -o xtrace
+argv0=ninja
+PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+
+# TODO: take aur-build flags (or command following `env`) as argument
```

Comment:
The ninja way here would be for you to generate the command-line and include it literally into the heredoc.

Reply:
This is slightly annoying with arrays, because inside a heredoc `"$@"` is expanded as `"$*"`. Luckily there's `@Q` since bash 5.